homework1/task02.py

```diff
@@ -0,0 +1,26 @@
+"""
+Given a cell with "it's a fib sequence" from slideshow,
+    please write function "check_fib", which accepts a Sequence of integers, and
+    returns if the given sequence is a Fibonacci sequence
+
+We guarantee, that the given sequence contain >= 0 integers inside.
+
+"""
+from typing import Sequence
+
+
+def check_fibonacci(data: Sequence[int]) -> bool:
+    if (
+        len(data) == 0
```

Comment:
I would argue that if the sequence contains no elements it is a Fibonacci sequence. You just can't disprove it, can you?

Reply:
Yes, I guess that empty sequence is a subsequence for every sequence, I will fix it

Reply:
Done